e2e/tests/graph.spec.ts

```
@@ -27,230 +26,229 @@ test.describe('Graph Tests', () => {
+        await graph.insertQuery("UNWIND range(1, 10) as x CREATE (n:n)-[e:e]->(m:m) RETURN *");
         await graph.clickRunQuery();
-        const initialGraph = await graph.getCanvasScaling();
-        await graph.clickOnZoomIn();
-        await graph.clickOnZoomIn();
-        const updatedGraph = await graph.getCanvasScaling();
-        expect(updatedGraph.scaleX).toBeGreaterThan(initialGraph.scaleX);
-        expect(updatedGraph.scaleY).toBeGreaterThan(initialGraph.scaleY);
-        await apicalls.removeGraph(graphName);
+        const nodes = await graph.getNodesGraphStats();
+        const edges = await graph.getEdgesGraphStats();
+        expect(parseInt(nodes ?? "")).toBe(20);
```

ESLint

[error] Missing radix parameter.

(radix)